Source/JavaScriptCore/wasm/WasmLLIntPlan.cpp

```diff
@@ -174,7 +174,7 @@ void LLIntPlan::didCompleteCompilation()
 
             LinkBuffer linkBuffer(jit, nullptr, LinkBuffer::Profile::Wasm, JITCompilationCanFail);
             if (UNLIKELY(linkBuffer.didFailToAllocate())) {
-                Base::fail(makeString("Out of executable memory in function entrypoint at index ", String::number(functionIndex)));
+                Base::fail(makeString("Out of executable memory in function entrypoint at index "_s, String::number(functionIndex)));
```

Comment:
Should say `functionIndex` instead of `String::number(functionIndex)`. Doesn’t need to be addressed in this patch, but I probably would.